docs/docs-components/Header.js

```diff
@@ -23,10 +23,10 @@ import trackButtonClick from './buttons/trackButtonClick.js';
 import { useNavigationContext } from './navigationContext.js';
 import { convertNamesForURL } from './DocsSideNavigation.js';
 
-const PAGE_HEADER_ZINDEX = new FixedZIndex(10);
-const ABOVE_PAGE_HEADER_ZINDEX = new CompositeZIndex([PAGE_HEADER_ZINDEX]);
-
 function Header() {
+  const PAGE_HEADER_ZINDEX = new FixedZIndex(10);
```

Comment:
These don't need to be defined within the component — any reason for moving them?